docs/_docs/configuration/menu-hjson.md

```diff
@@ -125,6 +125,7 @@ Many built in global/system methods exist. Below are a few. See [system_menu_met
 | `nextConf` | Sets the users message conference to the next available. |
 | `prevArea` | Sets the users message area to the previous available. |
 | `nextArea` | Sets the users message area to the next available. |
+| `setClientEncoding` | Sets the client encoding to either cp437 or utf-8. |
```

Comment:
Suggest instead of "either...", "such as..." or similar as other encoding are technically supported and hopefully some others down the line.

Reply:
Thanks, I was planning on changing this again in the future to add other ones (I'd really like to add conversions for at least PETSCII), but good point that others are already technically supported, I'll change the wording for now.

Reply:
iconv can be extended -- and in fact, I still need to patch it to properly convert UTF-8 to CP437 for *BBS purposes* where a couple control characters are currently stripped (where BBS's will display them generally)

Anyway, should be able to put in PETSCII mapping into iconv and make that avail.

Reply:
Changed. And definitely later additional encodings is something I'd love to tackle.